### ISSUE TITLE
Wrong conversion of triggers from file

### ISSUE THREAD

**Issue by dezdechado** [darold/ora2pg#203]
All body of trigger is converted into single line (ora2pg removes all `\n` and all user's comments, 
although adds own single-line comments, which breaks code).
Conversion from Oracle DB seems to work

**Comment by darold** [comment]
Yes, DML conversion from file need lot of additional work. The main development effort is done on export from Oracle, I don't know when it will be fixed.

Regards,
